catroid/src/main/java/org/catrobat/catroid/common/FileChecksumContainer.java

```diff
@@ -91,13 +93,13 @@ public int getUsage(String checksum) {
 	public void incrementUsage(String filepath) throws FileNotFoundException {
 		String checksum = null;
 		for (Map.Entry<String, FileInfo> entry : checksumFileInfoMap.entrySet()) {
-
 			if (entry.getValue().path.equalsIgnoreCase(filepath)) {
 				checksum = entry.getKey();
 				break;
 			}
 		}
 		if (checksum == null) {
+			Log.e(this.getClass().getSimpleName(), "incrementUsage of " + filepath);
```

Comment:
Create a log tag instead
```java
private static final String TAG = FileChecksumContainer.class.getSimpleName();
```
This would be consistent with all other source files, it looks cleaner and the next time someone wants to log something, s|he can use the TAG right away :wink:.

Maybe you can write a more meaningful log message to hint to the problem?